librz/type/type.c

```diff
@@ -1029,8 +1042,8 @@ RZ_API RZ_OWN char *rz_type_identifier_declaration_as_string(const RzTypeDB *typ
  * \brief Returns the type C identifier
  *
  * In case of the compound types it returns the name of identifier
- * For example, for "char **ptr" it will return "ptr",
- * for "const int **arr[56][76]" it will return "arr"
+ * For example, for "char **ptr" it will return "char",
+ * for "const int **arr[56][76]" it will return "int"
```

Comment:
It seems there's something wrong here.

Reply:
It isn't. It *was*.

Reply:
The identifier means the *type identifier* aka name, not the variable name.